extensions/zendesk/runtime/pom.xml

```diff
@@ -51,6 +51,12 @@
         <dependency>
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-zendesk</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>async-http-client</artifactId>
+                    <groupId>org.asynchttpclient</groupId>
+                </exclusion>
+            </exclusions>
```

Comment:
Why is this exclusion needed? If it there to solve dependency convergence issues, then I'd rather vote for managing `async-http-client` in our BOM. In that way, we can make sure that all our extensions (and tests) using it will use the same version which is better for our and end users' reproducibility.

Reply:
Hm... we already manage it in the BOM. Why is the exclusion needed then?

Reply:
I'll check, it is possible that it is not needed anymore - as it is in the BOM

Reply:
I forgot to remove this exclude during my last update of the PR - I'll fix it in a moment